deepmd/utils/data.py

```diff
@@ -269,6 +271,11 @@ def get_item_torch(self, index: int) -> dict:
         """
         i = bisect.bisect_right(self.prefix_sum, index)
         frames = self._load_set(self.dirs[i])
+        if self.multistru:
+            size = frames["coord"].shape[1]
+            rng = np.random.default_rng()
+            sample_idx = rng.integers(low=0, high=size, size=1)[0]
```

Comment:
Is it really random?

Reply:
We use the random generator in `deepmd.utils.rndom`. The generator here cannot be controlled.

Reply:
I have change it to 
`from deepmd.utils import random as dp_random`
`sample_idx = dp_random.choice(range(num_stru), size=1)[0]`
But I find that when num_stru = 11, the `sample_idx` is 1,4,8,10 all the time. I hope that the value is randomly selected from `range(0--10)`. How can I solve it?